db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_object_store_containers, :type, :string
```

Comment:
@durandom @kbrock thank you for the review. I've added following indeces:
```ruby
add_index :cloud_object_store_containers, :type
add_index :cloud_object_store_containers, [:ems_id, :type]
```
Will this do?

Also, I've prepared a dual PR that is the very same as this one, but for CloudObjectStoreObject. Could you please take a look while you're at it:
https://github.com/ManageIQ/manageiq/pull/14080